src/components/Drawer/MiniSideBar.tsx

```diff
@@ -0,0 +1,163 @@
+
+type RouteOption = {
+  label: string;
+  href: string;
+  icon: React.ReactNode;
+  disabled?: boolean;
+};
+
+const drawerWidth = 120;
+export default function MiniSideBar({ children }: PropsWithChildren) {
+  const dashboardRoutes: RouteOption[] = [
+    {
+      label: "Campaigns",
+      href: "/user/main/campaign",
+      icon: (
+        <CampaignOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+    // Possible future enhancements, not visible to user but help keep spacing
+    {
+      label: "Creatives",
+      href: "/user/main/creatives",
+      icon: (
+        <LightbulbOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+      disabled: true,
+    },
+    {
+      label: "Assets",
+      href: "/user/main/assets",
+      icon: (
+        <InsertPhotoOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+      disabled: true,
+    },
+    {
+      label: "Audiences",
+      href: "/user/main/audiences",
+      icon: (
+        <CampaignOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+      disabled: true,
+    },
+  ];
+
+  const settingsRoutes: RouteOption[] = [
+    {
+      label: "Account",
+      href: "/user/main/settings",
+      icon: (
+        <AccountBalanceOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+    {
+      label: "Profile",
+      href: "/user/main/profile",
+      icon: (
+        <AccountBoxOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+    {
+      label: "Support",
+      href: "mailto:selfserve@brave.com",
+      icon: (
+        <HeadsetMicOutlinedIcon
+          fontSize="large"
+          sx={{ color: "rgba(161, 171, 186, 1)" }}
+        />
+      ),
+    },
+  ];
+
+  return (
+    <Box sx={{ display: "flex" }}>
+      <CssBaseline />
+      <Drawer
+        variant="permanent"
+        open
+        sx={{
+          width: drawerWidth,
+        }}
+      >
+        <Toolbar />
+        <Box
+          sx={{
+            padding: "6px 0px 6px 0px",
+          }}
+        >
+          {dashboardRoutes.map((dr) => (
+            <ItemBox
+              href={dr.href}
+              icon={dr.icon}
+              label={dr.label}
+              disabled={dr.disabled}
+            />
+          ))}
+          <Divider sx={{ mt: 3, mb: 3 }} />
+          {settingsRoutes.map((sr) => (
+            <ItemBox href={sr.href} icon={sr.icon} label={sr.label} />
+          ))}
+        </Box>
+      </Drawer>
+      {children}
+    </Box>
+  );
+}
+
+const ItemBox = (props: RouteOption) => {
+  return (
+    <Box
+      display="flex"
+      component={Button}
+      href={props.href}
```

Comment:
URI's are hardcoded a few lines above, but let me know if something needs to be actioned here

Reply:
Since this is an in-app navigation, you shouldn't you use react-router's Link component to perform the navigation?  Navigating to an `href` will cause the SPA to reload.

Reply:
Have changed this to routerlink

Reply:
This is now doing react-router in-app navigation, I am going to mark this as resolved